lib/mvi/service.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 require 'savon'
+require_relative 'response'
+require_relative 'mock_service'
```

Comment:
Is the mock service used in the real service? I don't see any references to it in this file